include/mpd/async.h

```diff
@@ -209,6 +209,10 @@ mpd_malloc
 char *
 mpd_async_recv_line(struct mpd_async *async);
 
+
+struct mpd_binary
```

Comment:
Don't return a struct in C; it's bad style.

Reply:
Returns now a pointer to the struct.